CHANGELOG.rst

```diff
@@ -1,9 +1,24 @@
 .. _changelog:
 
+0.33.0
+------
+
+Renku ``0.33.0`` upgrades the PostgreSQL version that is shipped with the project.
+
+NOTE: make sure to check `helm-chart/values.yaml.changelog.md` for instructions on how to
```

Comment:
Who is the audience of this NOTE? The Renku admins or Renku users in general? I understand the former, and if so, the note may be confusing for regular users. Could we specify it? (e.g. Note for Renku Admins:...)